src/main/js/src/components/Alert/Alert.js

```diff
@@ -1,36 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Hyperlink from '../UI/Hyperlink/Hyperlink';
+import classes from './Alert.module.css';
 
 const alert = props => (
-  <div className={}>
-    <div
-      className={}
-    >
-      <div className={}>
-        <div className={}>{props.title}</div>
-        {props.optionalText && (
-          <div className={}>
-            {props.optionalText}
-          </div>
-        )}
+  <div className={classes.Alert}>
+    <div className={[classes.AlertContainer, classes.AlertError].join(' ')}>
+      <div className={classes.AlertTitle}>{props.title}</div>
+      {props.optionalText && (
+        <div className={classes.AlertText}>{props.optionalText}</div>
+      )}
+      {props.returnLinkTo && (
+        <div className={classes.AlertText}>
+          <Hyperlink to={props.returnLinkTo} text={props.returnLinkText} />
+        </div>
+      )}
+      {props.onClose && (
         <button
-          className={}
           type="button"
           title="Close"
           aria-label="Close"
           onClick={props.onClose}
         >
           <span aria-hidden="true">×</span>
         </button>
-      </div>
+      )}
     </div>
   </div>
 );
 
-alert.PropTypes = {
-  title: PropTypes.string,
+alert.propTypes = {
+  title: PropTypes.isRequired,
```

Comment:
This could have PropTypes.string.isRequired to indicate the expected type of prop.